skimage/feature/util.py

```diff
@@ -145,16 +146,29 @@ def plot_matched_features(
     ax.imshow(image, cmap='gray')
     ax.axis((0, image0.shape[1] + offset[1], image0.shape[0] + offset[0], 0))
 
-    rng = np.random.default_rng()
+    rng = np.random.default_rng(seed=0)
```

Comment:
There's a bit on [using at least 32 bits to seed](https://numpy.org/doc/stable/reference/random/bit_generators/index.html#:~:text=do%20not%20recommend%20using%20small%20seeds%20below%2032%20bits), but I think that is only relevant if we want get more than one random state. In this case, `seed=0` should be fine as I understand it.

Reply:
I tried to put in a 128-bit integer but this always catches my eyes scrolling over the code even though it is just a detail that should not really matter.

To me it also seemed that using a `seed=0` does not make a difference for our use-case:
> All of the BitGenerators in numpy delegate that task to [SeedSequence](https://numpy.org/doc/stable/reference/random/bit_generators/generated/numpy.random.SeedSequence.html#numpy.random.SeedSequence), which uses hashing techniques to ensure that even low-quality seeds generate high-quality initial states.